lib/screens/scanning_page/components/https_service.dart

```diff
@@ -55,7 +49,7 @@ class _HttppState extends State<Httpp> {
   }
 }
 
-const String postUrl = "http://192.168.40.44:8007/api/product";
+const String postUrl = "http://127.0.0.1:8007/api/product";
 Future<Product> _getdata(String? id) async {
   String url = postUrl + '/$id';
```

Comment:
@batsy3 this is a repeatition of code and could be done in one line for example String url = `http://addresss/api/product/${id}`